### PR TITLE
Installation instructions for privly-jetpack

### PR DESCRIPTION
**Pull request by hitesh96db**
#36

### CODE REVIEW

**Comment by irdan** [comment]
I think all of the relevant information is present in your changes.  I'm not sure if we should even mention the XUL extension since it will be phased out.  Maybe remove any mention of XUL in the developer guide since it's intended for newbies.  Then we would wait until XUL is deprecated before merging this.

File: pages/develop.md [hunk 1]
```diff
@@ -216,6 +218,27 @@ In that location, you will find your Firefox profile directories, e.g. 12a3bc4d.
...
+To develop this add-on, you will need:
+* Firefox version >= 38.0
+* jpm
+
+For jpm installation, follow the instructions as mentioned [here](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Installation).
+Firefox v38.0 and greater can be found [here](http://ftp.mozilla.org/pub/firefox/releases/).
+
+1) Get the [source code](https://github.com/privly/privly-jetpack) from GitHub. You can download it by cloning the repository,
+`git clone --recursive https://github.com/privly/privly-jetpack.git`.
+2) Build [Privly Applications](https://priv.ly/pages/develop.html#PrivlyApplications)
```
Comment (smcgregor) [inline pages/develop.md]: Should be `[Privly Applications](#PrivlyApplications)` since it is an anchor link.

**Comment by smcgregor** [comment]
You can deprecate the Xul version in these docs now, and remove most of its details. We are unlikely to accept pull requests on the Xul version at this point so there is not reason to help people get their development environment setup.

**Comment by hitesh96db** [comment]
Done. :+1:

**Comment by smcgregor** [comment]
You still need details stating something like: "We migrated the old Firefox extension from Xul to Jetpack. The Xul extension is deprecated so these instructions are for the newer Jetpack architecture."

**Comment by hitesh96db** [comment]
Rebased with a new commit.

**Pull request merged**